<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailDialog</class>
 <widget class="QDialog" name="DetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QFrame" name="powerConsumptionQFrame">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="powerConsumptionCaptionQLabel">
          <property name="text">
           <string>Power consumption:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="powerConsumptionQLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="computationTimeQFrame">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="computationTimeCaptionQLabel">
          <property name="text">
           <string>Computation time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="computationTimeQLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="platformUsageQFrame">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="platformUsageCaptionQLabel">
          <property name="text">
           <string>Platform usage:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="platformUsageQLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
